tool/since.dart

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2019, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:yaml/yaml.dart';
+
+import 'crawl.dart';
+
+main() async {
+  // Print for caching in since.yaml.
+//  for (var lint in registeredLints) {
+//    var since = await findSinceLinter(lint);
+//    if (since != null) {
+//      print('${lint.name}: $since');
+//    }
+//  }
```

Comment:
Did you intend to remove the commented out code? If not, consider expanding on the comment to explain why it's still here.

Reply:
It may come in handy.  Comment updated.  Thanks!